.gitignore

```diff
@@ -1,3 +1,4 @@
 _site
 .sass-cache
 .jekyll-metadata
+.DS_Store
```

Comment:
personally i'm against adding this to gitignore because I add things individually using `git add --patch` and avoid using git add -A. But we can add it in a separate pr